data/db/mil-sony.xml

```diff
@@ -7,6 +7,7 @@
         <compat>M39/1</compat>
         <compat>DKL</compat>
         <compat>T2</compat>
+        <compat>Leica M</compat>
```

Comment:
While surely there are adapters for M on E, Lensfun does not contain them in the main database.  To add them to the personal databases of owners of such adapters, there is the tool `lensfun-add-adapter`.  (We do make exceptions for extremely rare mounts like T2 because they don't clutter the lenses lists presented in image processing software.)